content/blog/2016/2016-12-15-declarative-pipeline-beta.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Declaring the Beta Release of Declarative Pipeline Syntax "
+tags:
+- pipeline
+- newfeatures
+- blueocean
+author: hrmpw
+---
+
+All the way back in September, Andrew Bayer presented a link:https://www.cloudbees.com/introducing-new-way-define-jenkins-pipelines[sneak peak] of Declarative Pipeline at Jenkins World and there was much rejoicing. Although it has been available in the update center since August, we have waited to recommend it for all of your pipeline needs while we work out the kinks and solidify the syntax. This week we released version 0.71 of the link:https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Model+Definition+Plugin[_Pipeline-Model-Defintion_] plugin and wanted to crown it as the official Beta version of the Declarative Pipeline Syntax.
```

Comment:
Same comment as another PR.  Can you split this into multiple lines?  I'll wait to give further feedback.